libr/anal/p/anal_mips_cs.c

```diff
@@ -758,32 +758,38 @@ static int analop(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *buf, int len,
 	}
 	op->id = insn->id;
 	opsize = op->size = insn->size;
+	op->refptr = 0;
 	switch (insn->id) {
 	case MIPS_INS_INVALID:
 		op->type = R_ANAL_OP_TYPE_ILL;
 		break;
 	case MIPS_INS_LB:
 	case MIPS_INS_LBU:
 	case MIPS_INS_LBUX:
+		op->refptr = 1;
```

Comment:
add /* fallthrough */ comment to avoid problems with static source analysis tools

Reply:
There is a test that failed. Could you give me some suggestions for modification?I have added / * fallthrough * / comment
Thinks

Reply:
Failing test is unrelated, it is kind of randomly occuring:
- https://builds.sr.ht/~xvilka/job/274076#task-test-0
- https://github.com/radareorg/radare2/issues/17395